addons/editor-dark-mode/addon.json

```diff
@@ -1059,14 +1085,14 @@
       "id": "experimentalDark",
       "description": "A dark theme by Maximouse based on a userstyle. Not experimental",
       "values": {
-        "page": "#263241",
-        "primary": "#4d97ff",
-        "highlightText": "#4d97ff",
-        "menuBar": "#4d97ff",
+        "page": "#2f2641",
+        "primary": "#9966ff",
+        "highlightText": "#ad7eff",
+        "menuBar": "#855cd6",
         "popup": "#000a",
         "activeTab": "#282828",
         "tab": "#202020",
-        "selector": "#252c37",
+        "selector": "#2b2537",
```

Comment:
I understand why you changed the `page` and `selector` colors - the dark blue doesn't look very good together with the purple accents. But the equivalent colors in vanilla Scratch will remain light blue, so I think it's better to choose one of the folllowing two options:

* Less saturated blue
  * page: `#2e3238`
  * selector: `#292d32`

* Dark gray
  * page: `#202020`
  * activeTab: `#333333`
  * tab: `#282828`
  * selector: `#333333`
  * selector2: `#333333`

See the comments above for `primary` and `highlightText`.